### ISSUE TITLE
How are updates handled? 

### ISSUE THREAD

**Issue by PhyxionNL** [exceptionless/Exceptionless#39]
I wasn't able to find this anywhere, I know this isn't really an issue but I really need to know this (would be nice if there was a FAQ on the wiki somewhere). We currently use an Exceptionless server (and client) build from a month/two months ago or so. 

I'd like to know how changes and updates are handled by Exceptionless, especially the following situations:
- 1-2 months old client with latest server.
- 1-2 months old server with latest client.
- old server and old client to latest server and client.

Basically, how are the updates handled and does the server still process data from the old clients (and viceversa)?
